backstopConfig.js

```diff
@@ -65,13 +65,13 @@ const config = {
       label: 'Card with data-qa_card',
       selectors: ['[data-qa="card"]'],
     },
-    {
-      ...basic,
-      label: 'Card with data-qa_card-hover',
-      selectors: ['[data-qa="card"]'],
-      hoverSelector: '[data-qa="card-hover"]',
-      postInteractionWait: 1000,
-    },
+    // {
+    //   ...basic,
+    //   label: 'Card with data-qa_card-hover',
+    //   selectors: ['[data-qa="card"]'],
+    //   hoverSelector: '[data-qa="card-hover"]',
+    //   postInteractionWait: 1000,
+    // },
```

Comment:
if this piece of code is not being used, then it is better to remove it.
Since it still will be in the git history, it won't be lost.